README.md

```diff
@@ -3,44 +3,116 @@ Some helpful snippets of code to automate the creation of Cluster-Stacks
+- `OS_CLOUD=${OS_CLOUD:-openstack}`: Name of the cloud in the clouds.yaml file
+- `CS_CCMLB=octavia-ovn`: Cloud controller manager load balancer type (octavia-ovn or octavia-amphora)
+
+#### Cluster Stack Settings
+- `CS_MAINVER=`: Kubernetes major.minor version (e.g., 1.32)
+- `CS_VERSION=`: Cluster stack template version (e.g., v1 or v0-sha.XXXXXXX)
+- `CS_CHANNEL=custom`: Update channel for ClusterStack
+- `CS_AUTO_SUBSCRIBE=false`: Whether to automatically subscribe to updates
+
+#### Workload Cluster Settings
+- `CL_PATCHVER=`: Full Kubernetes version (e.g., 1.32.3)
+- `CL_NAME=`: Cluster name
+- `CL_PODCIDR=172.16.0.0/18`: Pod CIDR range
+- `CL_SVCCIDR=10.96.0.0/14`: Service CIDR range
+- `CL_CTRLNODES=1`: Number of control plane nodes
+- `CL_WRKRNODES=1`: Number of worker nodes
+- `CL_WORKER_CLASS=default-worker`: Worker class used for machine deployments
+- `CL_LB_TYPE=octavia-ovn`: Load balancer type (depends on OpenStack environment)
 
 ## Scripts
 ### Once per management host
-* 00-bootstrap-vm-cs.sh: Install the needed software to be able to do
-  cluster management on this host. (Developed for Debian 12.)
+* `00-bootstrap-vm-cs.sh`: Install the needed software to be able to do
+  cluster management on this host. (Developed for Debian and Ubuntu.)
   This is only needed if you do not have the needed tools preinstalled.
-* 01-kind-cluster.sh: Create kind cluster
-* 02-deploy-capi.sh: Install ORC and CAPI.
-* 03-deploy-cso.sh: Install the Cluster Stack Operator.
+* `01-kind-cluster.sh`: Create kind cluster
+* `02-deploy-capi.sh`: Install ORC and CAPI.
+* `03-deploy-cso.sh`: Install the Cluster Stack Operator.
 
 ### Once per OpenStack Project in which we want to install clusters (NS)
-* 04-cloud-secret.sh: Create namespace and secrets to work with the
-  wanted OpenStack project.
+* `04-cloud-secret.sh`: Create namespace and secrets to work with the
+  wanted OpenStack project. Uses the `CS_CCMLB` setting to configure
+  the cloud controller manager load balancer type.
 
 ### Once per Kubernetes aka CS version (maj.min)
-* 05-deploy-cstack.sh: Create the Cluster Stack which is a template
+* `05-deploy-cstack.sh`: Create the Cluster Stack which is a template
   for various clusters with the same major minor version of k8s.
   Should trigger cluster class creation and image registration.
-* 06-wait-clusterclass.sh: Wait for the cluster class (not yet implemented)
+* `06-wait-clusterclass.sh`: Wait for the cluster class to be ready
 
 ### Once per cluster
-* 07-create-cluster.sh: Create a workload cluster as per all the settings
+* `07-create-cluster.sh`: Create a workload cluster as per all the settings
   that are passed.
-* 08-wait-cluster.sh: Wait for the workload cluster (not yet implemented)
-
-* 17-delete-cluster.sh: Remove cluster again.
+* `08-wait-cluster.sh`: Check cluster status and save kubeconfig if ready.
+  This script no longer waits in a loop but provides immediate status
```

Comment:
It never did.
But that's what's expected from a script with the name wait.
So call it poll or implement waiting ...